rocAL/rocAL_pybind/amd/rocal/plugin/pytorch.py

```diff
@@ -98,32 +124,41 @@ def __next__(self):
                         img = output
                         draw_patches(img[i], i, self.bb_2d_numpy)
 
-
-            max_cols = max([len(row) for batch in self.bbox_list for row in batch])
+            max_cols = max([len(row)
```

Comment:
@fiona-gladwin why add the extra line?

Reply:
The line is removed and line 126 is present to separate from the for loop block